.gitmodules

```diff
@@ -149,6 +149,10 @@
 [submodule "third_party/mimalloc"]
 	path = third_party/mimalloc
 	url = https://github.com/microsoft/mimalloc.git
+[submodule "third_party/composable_kernel"]
+	path = third_party/composable_kernel
```

Comment:
nit: Maybe call it third_party/rocm_composable_kernel ? I think the name reads overly generic without a reference to rocm or AMD, and most people are not yet familiar with the lib.

Reply:
I think that would be a bit redundant, since the repo link points to ROCm